CONTRIBUTING.rdoc

```diff
@@ -32,27 +32,99 @@ To run commands like `gem install` from the repo:
-committers.  They are triaged with each new feature release and either move to
-the new version numbered milestone, left in "Future" or moved to "Unfulfilled
-Promises and Broken Dreams".  They may be closed after one year.
-
-Issues with accepted status in a feature release milestone have been reviewed
-and triaged and are scheduled for a fix or implementation.
-
-Issues with the feedback status may be closed one week after a request for more
-information from a collaborator.  They will be reopened when more information
-becomes available.
-
+RubyGems uses labels to track all issues and pull requests. In order to provide
+guidance to the community this is documentation of how labels are used in the
+rubygems repository.
+
+=== Type
+
+Most Issues or pull requests will have one of these labels, which describes the
+type of the issue or pull request.
+
+* <b>bug report</b> - An issue describing a bug in rubygems. This would be something
+  that is broken, confusing, unexpected behavior etc.
+* <b>bugfix</b> - A pull request that fixes a bug report.
+* <b>feature request</b> - An issue describing a request for a new feature or
+  enhancement.
+* <b>feature implementation</b> - A pull request implementing a feature request.
+* <b>question</b> - An issue that is a more of a question than a call for specific
+  changes in the codebase.
+* <b>cleanup</b> - Generally for a pull request that improves the code base without
+  fixing a bug or implementing a feature.
+* <b>major bump</b> - This issue or pull request requires a major version bump
+* <b>administrative</b> - This issue relates to adminstrative tasks that need to
+  take place as it relates to rubygems
+
+Bug report and Bugfix have the same color. And Feature implementation and
+feature request have the same color since they are related labels.
+
+=== Workflow
+
+These are labels that indicate the state of an issue, where it is in the process
+from being submitted to being closed. These are listed in rough progression
+order from submitted to closed.
+
+* <b>triage</b> - This is an issue or pull request that needs to be properly
+  labeled by by a maintainer.
+* <b>accepted</b> - This issue / pull request has been accepted as valid and
+  will be worked on by someone.
+* <b>ready for work</b> - An issue that is available for collaboration. This issue
+  should have existing discussion on the problem, and a description of how to go
+  about solving it. This label should be removed once someone has said they are
+  going to work on it.
+* <b>claimed</b> - An issue that is claimed by a member of the community and is
+  working on it. If the member can be assigned to the issue, they should be.
+* <b>maintainer feedback</b>- This issue/pull request is waiting on feedback from
+  one of the rubygems maintainers, in this case, it should also be assigned to
+  that maintainer.
+* <b>collaborator feedback</b> - This issue/pull request is waiting on feedback from
```

Comment:
@segiddins I'd be fine consolidating feedback to one tag again and just @ folks for if they are the ones that need to take care of it. make it simpler.